sig-storage/annual-report-2024.md

```diff
@@ -30,21 +54,23 @@
       Please raise an issue in kubernetes/community, if the KEP metadata is correct but the generated list is incorrect.
 -->
 
-<!-- 
   - Alpha
-    - [3314 - KEP Template](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/3314-csi-changed-block-tracking) - v1.32
-    - [4049 - Storage Capacity Scoring of Nodes for Dynamic Provisioning](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/4049-storage-capacity-scoring-of-nodes-for-dynamic-provisioning) - v1.32
+    - [1710 - Speed up recursive SELinux label change](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1710-selinux-relabeling)
+      - SELinuxMount moved to Alpha in v1.30
+      - SELinuxChangePolicy moved to Alpha in v1.32
 
   - Beta
     - [1790 - Recover from volume expansion failure](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1790-recover-resize-failure) - v1.32
     - [2644 - Honor Persistent Volume Reclaim Policy](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/2644-honor-pv-reclaim-policy) - v1.31
     - [3476 - Volume Group Snapshot](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/3476-volume-group-snapshot) - v1.32
-    - [3751 - Kubernetes Volume Provisioned IO](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/3751-volume-attributes-class) - v1.31
+    - [3751 - VolumeAttributesClass ModifyVolume](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/3751-volume-attributes-class) - v1.31
+    - [2589 - Portworx in-tree to CSI driver migration](https://github.com/kubernetes/enhancements/issues/2589) - on-by-default in v1.31
 
   - Stable
     - [3141 - Prevent unauthorised volume mode conversion](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/3141-prevent-volume-mode-conversion) - v1.30
     - [3756 - Robust VolumeManager reconstruction after kubelet restart](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/3756-volume-reconstruction) - v1.30
-    - [3762 - PersistentVolume last phase transition time](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/3762-persistent-volume-last-phase-transition-time) - v1.31 -->
+    - [3762 - PersistentVolume last phase transition time](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/3762-persistent-volume-last-phase-transition-time) - v1.31
+    - [1847 - Auto remove PVCs created by StatefulSet](https://github.com/kubernetes/enhancements/issues/1847) - v1.32 (co-owned with SIG-Apps)
 
 ## [Subprojects](https://git.k8s.io/community/sig-storage#subprojects)
```

Comment:
I go through the sub projects and find 
- https://github.com/kubernetes-csi/cluster-driver-registrar and https://github.com/kubernetes-csi/driver-registrar are deprecated, should we archive them?

Reply:
Good question.  Let me bring this up in the next SIG Storage meeting.

Reply:
Discussed at SIG Storage meeting.  We can archive them.  I'll submit a request.

Reply:
PR submitted: https://github.com/kubernetes/org/issues/5450